source/functions/initializeWecSim.m

```diff
@@ -511,11 +511,25 @@
 for iW = 1:length(waves)
     waves(iW).listInfo();
 end; clear iW
-fprintf('\nList of Body: ');
-fprintf('Number of Bodies = %u \n',simu.numHydroBodies)
-for i = 1:simu.numHydroBodies
-    body(i).listInfo
+fprintf('\nList of Body:\n ');
+fprintf('Number of Hydro Bodies = %u \n',simu.numHydroBodies)
+i = 1; 
+while i <= simu.numHydroBodies
```

Comment:
To be consistent with the rest of the source code, I think we should use for loops. "for i = 1:simu.numHydroBodies"

Reply:
I applied your recommendations.
I also made the code print the non-hydro body details same as the hydro body